### PR TITLE
Add samples for reading and writing packages using the API

### PR DESCRIPTION
**Pull request by joelverhagen**
Address https://github.com/NuGet/docs.microsoft.com-nuget/issues/2094
Depends on https://github.com/NuGet/Samples/pull/40

### CODE REVIEW

**Comment by anangaur** [comment]
Docs Build status updates of commit _[473f576](https://github.com/NuGet/docs.microsoft.com-nuget/commits/473f5765d3c2c7c944929862657f3f2ff1f55ec7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008131646258561-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4x%2FxiBbDwisZQtyiKSKFOzAK7XxcD2059LaBMnA8QnA%3D&st=2020-08-13T16%3A41%3A58Z&se=2020-09-13T16%3A46%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[c40d1f0](https://github.com/NuGet/docs.microsoft.com-nuget/commits/c40d1f045bd965988100368056e65328a86994a5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008131713181203-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3TQ0yrsmtGc%2FemxdI0XrCiaKZYua27vgRjSP1oNjKCk%3D&st=2020-08-13T17%3A08%3A50Z&se=2020-09-13T17%3A13%3A50Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[255b991](https://github.com/NuGet/docs.microsoft.com-nuget/commits/255b991cf08df5dd821fc44412fc90cefc82cc62)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008131716060259-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ykdLn%2BdPg5z40FdtHyAGbUPnJdMWodVkvHnajS%2BYNG4%3D&st=2020-08-13T17%3A11%3A48Z&se=2020-09-13T17%3A16%3A48Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/reference/NuGet-Client-SDK.md [hunk 1]
```diff
@@ -55,6 +57,27 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
...
+> [!IMPORTANT]
+> It is strongly recommended that NuGet packages are created using the official NuGet tooling and **not** using this
+> low level API. There are a variety of characteristics important for a well-formed package and the latest version of
+> tooling helps incorporate these best practices.
+> 
+> For more information about creating NuGet packages, see the overview of the
+> [package creation workflow](../create-packages/overview-and-workflow.md) and the documentation for official pack
+> tooling (e.g. [using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
+
+[!code-csharp[CreatePackage](~/../nuget-samples/NuGetProtocolSamples/Program.cs?name=CreatePackage)]
```
Comment (Mikejo5000) [inline docs/reference/NuGet-Client-SDK.md:73]: The sample isn't rendering. Path issue?

File: docs/reference/NuGet-Client-SDK.md [hunk 2]
```diff
@@ -55,6 +57,27 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
...
+
+### Read a package
+
+Read a package from a file stream using [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging).
+
+[!code-csharp[ReadPackage](~/../nuget-samples/NuGetProtocolSamples/Program.cs?name=ReadPackage)]
```
Comment (Mikejo5000) [inline docs/reference/NuGet-Client-SDK.md:79]: Sample not rendering

File: docs/reference/NuGet-Client-SDK.md [hunk 3]
```diff
@@ -55,6 +57,27 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
 
 [!code-csharp[SearchPackages](~/../nuget-samples/NuGetProtocolSamples/Program.cs?name=SearchPackages)]
 
+### Create a package
+
+Create a package, set metadata, and add dependencies using [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging).
+
+> [!IMPORTANT]
+> It is strongly recommended that NuGet packages are created using the official NuGet tooling and **not** using this
+> low level API. There are a variety of characteristics important for a well-formed package and the latest version of
```
Comment (Mikejo5000) [inline docs/reference/NuGet-Client-SDK.md:66]:
```suggestion
> low-level API. There are a variety of characteristics important for a well-formed package and the latest version of
```

File: docs/reference/NuGet-Client-SDK.md [hunk 4]
```diff
@@ -55,6 +57,27 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
...
+> tooling helps incorporate these best practices.
+> 
+> For more information about creating NuGet packages, see the overview of the
+> [package creation workflow](../create-packages/overview-and-workflow.md) and the documentation for official pack
+> tooling (e.g. [using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
```
Comment (Mikejo5000) [inline docs/reference/NuGet-Client-SDK.md:71]:
```suggestion
> tooling (For example, [Using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
```

File: docs/reference/NuGet-Client-SDK.md (hunk 1 above)
Comment (joelverhagen) [reply docs/reference/NuGet-Client-SDK.md:73]: This PR needs to be completed first:
https://github.com/NuGet/Samples/pull/40

File: docs/reference/NuGet-Client-SDK.md (hunk 2 above)
Comment (joelverhagen) [reply docs/reference/NuGet-Client-SDK.md:79]: This PR needs to be completed first:
https://github.com/NuGet/Samples/pull/40

File: docs/reference/NuGet-Client-SDK.md [hunk 5]
```diff
@@ -21,10 +21,12 @@ You can find the source code for these packages in the [NuGet/NuGet.Client](http
 
 ## Getting started
 
-### Install the package
+### Install the packages
 
 ```ps1
-dotnet add package NuGet.Protocol
+dotnet add package NuGet.Protocol  # interact with HTTP and folder based NuGet package feeds, includes NuGet.Packaging
```
Comment (Mikejo5000) [inline docs/reference/NuGet-Client-SDK.md:27]:
```suggestion
dotnet add package NuGet.Protocol  # interact with HTTP and folder-based NuGet package feeds, includes NuGet.Packaging
```

**Comment by anangaur** [comment]
Docs Build status updates of commit _[1daefd4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/1daefd49a077af88e8985ae48fdcb441761c94cb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008132113118962-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=P4Q8KrOgJMLBZF9IT8A9luvDHmVP%2Bv7T36nCHL8hPFg%3D&st=2020-08-13T21%3A08%3A44Z&se=2020-09-13T21%3A13%3A44Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[b4812f5](https://github.com/NuGet/docs.microsoft.com-nuget/commits/b4812f52862b8a6bd56a33edebde85819374dad9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008132113198820-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=MaCtcvXjA5yXJdi5JB9kqVM%2BWD2JQ9WIRV0yAuA5%2FIw%3D&st=2020-08-13T21%3A09%3A28Z&se=2020-09-13T21%3A14%3A28Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[36611f2](https://github.com/NuGet/docs.microsoft.com-nuget/commits/36611f2672fab94b456f4df619a6c1f53d7ffe21)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-f406f003)

<a id="f406f003"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008132159224237-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=eFCCRdxfYZizALFDBW%2BUGsiFqdG1SBzVsmsYhDe6mU8%3D&st=2020-08-13T21%3A54%3A59Z&se=2020-09-13T21%3A59%3A59Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/reference/NuGet-Client-SDK.md [hunk 6]
```diff
@@ -55,6 +57,27 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
...
+Create a package, set metadata, and add dependencies using [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging).
+
+> [!IMPORTANT]
+> It is strongly recommended that NuGet packages are created using the official NuGet tooling and **not** using this
+> low-level API. There are a variety of characteristics important for a well-formed package and the latest version of
+> tooling helps incorporate these best practices.
+> 
+> For more information about creating NuGet packages, see the overview of the
+> [package creation workflow](../create-packages/overview-and-workflow.md) and the documentation for official pack
+> tooling (For example, [Using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
```
Comment (loic-sharma) [inline docs/reference/NuGet-Client-SDK.md:71]: The upper casing seems odd to me. What do you think of this?

```suggestion
> tooling (for example, [using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
```

I feel less strongly about `Using the dotnet CLI`, feel free to keep that as-is.

**Comment by anangaur** [comment]
Docs Build status updates of commit _[36611f2](https://github.com/NuGet/docs.microsoft.com-nuget/commits/36611f2672fab94b456f4df619a6c1f53d7ffe21)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008171633183409-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=v59DEHjMw9ZoIBVJntKR88VuqyuQ3hl9S9m%2BjIV4Ui4%3D&st=2020-08-17T16%3A28%3A53Z&se=2020-09-17T16%3A33%3A53Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/reference/NuGet-Client-SDK.md (hunk 6 above)
Comment (Mikejo5000) [reply docs/reference/NuGet-Client-SDK.md:71]: @loic-sharma You're right about the "for example". The reference to another topic ("Using the...") is capitalized, just like the topic.

File: docs/reference/NuGet-Client-SDK.md (hunk 1 above)
Comment (joelverhagen) [reply docs/reference/NuGet-Client-SDK.md:73]: This should be fixed now.

**Comment by anangaur** [comment]
Docs Build status updates of commit _[adad2b8](https://github.com/NuGet/docs.microsoft.com-nuget/commits/adad2b8d4c7255e5361940ad229190a55d8df52f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2094/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-2126) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008171636400453-2126%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=MOEavEyQDssT3lXW%2FF5hHqIqxPCuhAspZCcaowi6gPs%3D&st=2020-08-17T16%3A32%3A20Z&se=2020-09-17T16%3A37%3A20Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**